src/SignalRServiceExtension/TriggerBindings/ServerlessHub.cs

```diff
@@ -27,22 +29,26 @@ public abstract class ServerlessHub : IDisposable
     {
         private static readonly Lazy<JwtSecurityTokenHandler> JwtSecurityTokenHandler = new Lazy<JwtSecurityTokenHandler>(() => new JwtSecurityTokenHandler());
         private bool _disposed;
-        private readonly IServiceManager _serviceManager;
+        private readonly IServiceContext _serviceContext;
```

Comment:
remove `_`

Reply:
The coding convention throughout the project is not consistent. Sometimes I just follow the convention according to the old file. I think another PR to fix this may be better.